integrations/clickup/CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
+
+# Port_Ocean 0.1.2 (2024-07-04)
+
+### Features
+
+- content goes here (0.1.0)
+- created blueprint and port-app-config (0.1.0)
+- get teams, projects and issues (0.1.1)
+- edited and updated port-app-config (0.1.2)
+- edited client.py to put team id in project list (0.1.2)
+
+### Bug Fixes
+
+- lint code (0.1.2)
+
+
+# Port_Ocean 0.1.2 (2024-07-04)
+
+### Features
+
+- content goes here (0.1.0)
+- created blueprint and port-app-config (0.1.0)
+- get teams, projects and issues (0.1.1)
+- edited and updated port-app-config (0.1.2)
+- edited client.py to put team id in project list (0.1.2)
+
+
+# Port_Ocean 0.1.1 (2024-07-04)
+
+### Features
+
+- content goes here (0.1.0)
+- created blueprint and port-app-config (0.1.0)
+- get teams, projects and issues (0.1.1)
+
+
+# Port_Ocean 0.1.0 (2024-07-04)
+
```

Comment:
This could be miscommunication but we don't generate changelog for every commit/changes you make. Change is based on PR. Since this is the first PR, you should have something like

# Port_Ocean 0.1.1 (2024-07-04)

### Features

- Implemented Clickup integration to ingest teams, projects and issues